4week/src/pages/LpDetailPage.tsx

```
@@ -1,300 +1,316 @@
+        onChange={handleImageChange}
+        className="hidden"
+        accept="image/*"
+      />
+
+      <div className="flex flex-col gap-4">
+        {/* 작성자 / 날짜 */}
+        <div className="flex justify-between">
+          <h1>
+            {typeof lp.data.author === "string"
+              ? lp.data.author
+              : (lp.data.author as any)?.name}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)